projects/email_alert_notifications/templates/email_alert_s3_bucket_policy.json

```diff
@@ -0,0 +1,32 @@
+{
+  "Version": "2012-10-17",
+  "Id": "Policy1460992104238",
```

Comment:
Fixed in ca7c3c1. Seems that using a `uuid` is the way to go although I haven't used an `ID` at all in the other policies. I'm not sure if there's any benefit to using them. As it happens this policy confuses `terraform apply` into thinking it has changed every time it is run which I hoped the change you suggested might fix but didn't unfortunately. I think it's line 27 not being specific enough but haven't been able to work out a more 'fully qualified' version that works.